cfgov/prepaid_agreements/views.py

```diff
@@ -1,9 +1,17 @@
+from six.moves.urllib.parse import urlparse
+
 from django.contrib.postgres.search import SearchVector
 from django.core.paginator import InvalidPage, Paginator
 from django.db.models import Q
 from django.shortcuts import get_object_or_404, render
+from django.urls import reverse
 
 from prepaid_agreements.models import PrepaidProduct
+from v1.models.snippets import ReusableText
+
+
+DISCLAIMER_SNIPPET_TITLE = 'Prepaid agreements database disclaimer'
```

Comment:
See comment below, but instead of defining these as variables, could you do
`DISCLAIMER_TEXT = ReusableText.objects.filter(title='Prepaid agreements database disclaimer').first()` and so on?

Reply:
Good point! Updated in https://github.com/cfpb/cfgov-refresh/pull/5239/commits/9574f04596eea8b1ac2630af377e96387c8f5bf5